Philips.CodeAnalysis.Test/Verifiers/CodeFixVerifier.cs

```diff
@@ -113,6 +115,9 @@ private void VerifyFix(DiagnosticAnalyzer analyzer, CodeFixProvider codeFixProvi
 				}
 			}
 
+			//after applying all of the code fixes, there shouldn't be any problems remaining
+			Assert.IsTrue(allowNewCompilerDiagnostics || !analyzerDiagnostics.Any(), $@"After applying the fix, there still exists {analyzerDiagnostics.Length} diagnostic(s), including {analyzerDiagnostics.FirstOrDefault()?.Id}");
```

Comment:
Replace the `analyzerDiagnostics.FirstOfDefault()?.Id` with `string.Join(",", analyzerDiagnostics.Select(diag => diag.Id)` to make sure we show all the diagnostics

Reply:
done

Reply:
I don't see the code change

Reply:
Never mind, GitHub played tricks on me